### PR TITLE
Add link for issue tracker for no results

### LINKED ISSUE #308: Add info for bug reports to pages without search results
At times there are bugs of the service and there are no results. Bug reports can be submitted to the issue tracker. Please add this info to the page, if no results are shown.

screenshot from 2017-03-30 14-06-48

After the following please add the sentence below:

> We will crawl the social web with this search term again in the upcoming hours.
Please check back again later. In the meantime please try the following to get relevant results.
> Make sure that all words are spelled correctly.
Try different keywords, accounts or hashtags.
Try more general keywords, accounts or hashtags.

If you encountered an error of the system, please submit a bug report in the [loklak issue tracker here](https://github.com/fossasia/loklak_search/issues/).

### PR DESCRIPTION
**Pull request by hemantjadon**
Link to the loklak issue tracker added to the "No results found page"

**Screenshots (if appropriate)** 
screen shot 2017-08-01 at 10 56 22 am

**Closes #308**

### CODE REVIEW

File: src/app/feed/feed-not-found/feed-not-found.component.html [hunk 1]
```diff
@@ -11,5 +11,10 @@
 			<li>Try different keywords, accounts or hashtags.</li>
 			<li>Try more general keywords, accounts or hashtags.</li>
 		</ul>
+
+		<p>
+			If you encountered an error of the system, please submit a bug report in the
```
Comment (singhpratyush) [inline src/app/feed/feed-not-found/feed-not-found.component.html]: How about something this?

`If the problem persists, please submit a bug report to the loklak issue tracker.`

Comment (vibhcool) [inline src/app/feed/feed-not-found/feed-not-found.component.html]: the existing line seems better

**Comment by hemantjadon** [comment]
@Achint08 it is already linked

File: src/app/feed/feed-not-found/feed-not-found.component.html (hunk 1 above)
Comment (hemantjadon) [inline src/app/feed/feed-not-found/feed-not-found.component.html]: I guess the existing one is better

**Pull request merged**